bulletin-buddy-post-service/src/main/java/com/github/bulletinboard/models/Post.java

```diff
@@ -5,25 +5,36 @@
 import java.util.Date;
 import java.util.UUID;
 
-public class Offer {
+public class Post {
     private UUID id;
     private String title;
     private String body;
     private String publisherEmail;
 
+    // add Enumerated
```

Comment:
Should be internal to Offer or can it be extracted? I'm wondering if we'll need to use this enum elsewhere....